charactersheet/charactersheet/services/character/proficiency_service.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var ProficiencyService = new SharedServiceManager(_ProficiencyService, {});
+
+
+function _ProficiencyService(configuration) {
+    var self = this;
+
+    self.proficiency = ko.observable();
+    self.characterId = ko.observable();
+
+    self.init = function() {
+        Notifications.profile.changed.add(self.dataHasChanged);
+        Notifications.stats.changed.add(self.dataHasChanged);
+
+        // Kick it off the first time.
+        self.characterId(CharacterManager.activeCharacter().key());
+        self.dataHasChanged();
+    };
+
+    self.dataHasChanged = function() {
+        // Starts at one because of the static one in the calculation
+        var proficiency = 1;
```

Comment:
I'd suggest init this to 0. Then below in the levelBonus method, return Math.ceil(level / 4) + 1 just like the text in the popover.  This is more clear I think and doesn't require this comment.